app/controllers/api/users_controller.rb

```diff
@@ -13,8 +13,28 @@ def show
     authorize @user, policy_class: Api::UsersPolicy
   end
 
```

Comment:
The `sign_in` method updates the user's sign-in information twice. The first update in `custom_token_initialize_values` is redundant and can be removed to improve performance.